hooks/test/test_test.go

```diff
@@ -71,3 +71,14 @@ func TestLoggingWithHooksRace(t *testing.T) {
 	entries := hook.AllEntries()
 	assert.Equal(100, len(entries))
 }
+
+func TestFatalWithPanic(t *testing.T) {
+	assert := assert.New(t)
+
+	logger, hook := NewNullLogger(FatalPanics)
```

Comment:
We can just call FatalPanics on the returned logger object rather than implementing that with a callback.